cmd/convox/services.go

```diff
@@ -298,6 +305,32 @@ func cmdServiceInfo(c *cli.Context) error {
 	return nil
 }
 
+func cmdServiceURL(c *cli.Context) error {
+	if len(c.Args()) != 1 {
+		stdcli.Usage(c, "url")
+		return nil
+	}
+
+	name := c.Args()[0]
+
+	service, err := rackClient(c).GetService(name)
+	if err != nil {
+		return stdcli.ExitError(err)
+	}
+
+	if service.Status == "failed" {
+		fmt.Printf("Reason  %s\n", service.StatusReason)
```

Comment:
Probably make this an error of some sort with `stdcli.ExitError()` since there won't be a valid URL to show.